doc/rst/conf.py

```diff
@@ -133,6 +133,7 @@ def setup(app):
                     'builtins/String.rst',
                     'modules/standard/AutoMath.rst',
                     'modules/standard/ChapelIO.rst',
+                    'usingchapel/editors/*.rst',
```

Comment:
The comment says "directories to ignore", so I think you could just say `usingchapel/editors`